implementations/rust/ockam/ockam_command/tests/bats/local/nodes.bats

```diff
@@ -162,3 +162,14 @@ force_kill_node() {
   run_failure "$OCKAM" node create --node-config "{name: n}" --variable MY_VAR=
   assert_output --partial "Empty value for variable 'MY_VAR'"
 }
+
+@test "node - http server is started by default" {
```

Comment:
I would phrase this more in terms of the provided feature: "A node provides an HTTP endpoint to query its status".
And maybe "The node show command displays the address to use in order to query the HTTP endpoint" as a comment for testing the `http_server` field.
Another question is: since you wrote "started by default", is there a way to avoid that behavior? If not, then we should remove "by default".